### PR TITLE
Metadata refresh, some minor updates, removal of ocaml 4.13.1

### PR DESCRIPTION
**Pull request by psafont**
The metadata updates need to be tested before merging, they changed where the files are hosted. Maybe we can do the same for our patches? It will deduplicate the patches we have for rpclib

As always, it's better to review the commits one by one.

### CODE REVIEW

**Comment by psafont** [comment]
The current failure is because of a test in xapi not being attached to any package, fixed by: https://github.com/xapi-project/xen-api/pull/5684

**Review by lindig — APPROVED** [review]

File: packages/ocaml/ocaml-base-compiler.4.13.1/opam [hunk 1]
```diff
@@ -1,43 +1,94 @@
 opam-version: "2.0"
```
Comment (edwintorok) [inline packages/ocaml/ocaml-base-compiler.4.13.1/opam]: Do we still need 4.13.1 here? Is it for LCM?

File: packages/ocaml/ocaml-base-compiler.4.14.1/opam [hunk 2]
```diff
@@ -2,7 +2,10 @@ opam-version: "2.0"
 name: "ocaml-base-compiler"
```
Comment (edwintorok) [inline packages/ocaml/ocaml-base-compiler.4.14.1/opam:2]: Aren't we on 4.14.2? Should we drop 4.14.1?

**Review by edwintorok — APPROVED** [review]
Minor comment about dropping 4.14.1

File: packages/ocaml/ocaml-base-compiler.4.14.1/opam (hunk 2 above)
Comment (psafont) [reply packages/ocaml/ocaml-base-compiler.4.14.1/opam:2]: Yes, eventually, some people might not have updated their opam switches to ocaml 4.14.2, which means bad things will happen when the version is removed.

**Comment by psafont** [comment]
This is being tested in suite run 201807

**Pull request merged**